pkg/errors/errors.go

```diff
@@ -0,0 +1,44 @@
+package errors
+
+import "fmt"
+
+const (
+	// Value for destinguishing non-codedError type.
+	// Not used by codedError.
+	NotCodedErrorType = iota - 1
```

Comment:
if they all suffix with `Error` , this one should do the same.

Reply:
Also Error codes should be split from the coded error object, as we may reuse it with different error codes in other porjects.

Reply:
> if they all suffix with `Error` , this one should do the same.

As I described in comment, this is not actually error. This is why it doesn't have suffix and, IMHO, with suffix it'll be confusing what this "Error" mean.

Reply:
how about: `NoCodedError`  ? this reads as there is no coded error but is still suffixed with `Error` ?

Reply:
Yes, it reads as there is an error "no coded")

Reply:
if we split the error codes from the implementation and let the consumer of this package define the codes then you will be left with just one constant you need which is the special value `NotCodedErrorType`

What do you think ?